go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/mattn/goveralls v0.0.2
 	github.com/pborman/uuid v1.2.0 // indirect
 	github.com/quasilyte/go-consistent v0.0.0-20190521200055-c6f3937de18c
-	github.com/quasilyte/go-ruleguard v0.3.1
+	github.com/quasilyte/go-ruleguard v0.3.2
 	github.com/quasilyte/go-ruleguard/dsl v0.3.1
```

Comment:
How is this version supposed to be updated?
```bash
$ go get github.com/quasilyte/go-ruleguard/dsl@v0.3.2
go get github.com/quasilyte/go-ruleguard/dsl@v0.3.2: github.com/quasilyte/go-ruleguard/dsl@v0.3.2: invalid version: unknown revision dsl/v0.3.2
```

Reply:
Should you? :) Looks like there is no `dsl` release tagged `v0.3.2` so it's safe to leave it as it's.

FYI 'how to release submodule?` - there should be a tag `dsl/v0.3.2` in ruleguard repo.